src/src/cron/cron.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
+import { UserCreationService } from './usercreation.service';
+import { FaceIndexingService } from './faceIndexing.service';
+import { MarkAttendanceService } from './markAttendance.service';
+import { AwsRekognitionModule } from '../services/aws-rekognition/aws-rekognition.module';
+import { HasuraModule } from '../services/hasura/hasura.module';
+@Module({
+	imports: [AwsRekognitionModule, HasuraModule],
+	providers: [
+		UserCreationService,
```

Comment:
lets rename these services as

usercreation.service.ts => faUserIndexing.cron.ts
faceIndexing.service.ts => faFaceIndexing.cron.ts
markAttendance.service.ts => faAttendanceProcessing.cron.ts

UserCreationService => FaUserIndexingCron
FaceIndexingService => FaFaceIndexingCron
MarkAttendanceService => FaAttendanceProcessingCron

Reply:
I renamed all as you suggested.